docs/agents/influxdb_publisher_v2.rst

```diff
@@ -0,0 +1,152 @@
+.. highlight:: rst
+
+.. _influxdb_publisher_v2:
+
+========================
+InfluxDB Publisher v2 Agent
+========================
+
+The InfluxDB Publisher v2 Agent is an upgraded InfluxDB Publisher Agent that
+uses the influxdb-client package instead of influxdb package to work with
+InfluxDB v2.
+
+.. argparse::
+   :module: ocs.agents.influxdb_publisher_v2.agent
+   :func: make_parser
+   :prog: agent.py
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container. Also included is an example for setting up
+Grafana to display data from InfluxDB.
+
+OCS Site Config
+```````````````
+
+Add an InfluxDBAgent to your OCS configuration file::
+
+      {'agent-class': 'InfluxDBAgentv2',
+       'instance-id': 'influxagent',
+       'arguments': [['--initial-state', 'record'],
+                     ['--org', 'socs'],
+                     ['--database', 'ocs_feeds']]},
+
+Docker Compose
+``````````````
+
+Add the InfluxDB Publisher v2 Agent container to your docker-compose file::
+
+  ocs-influxdb-publisher-v2:
+    image: simonsobs/ocs:latest
+    hostname: ocs-docker
+    environment:
+      - INSTANCE_ID=influxagent
+    volumes:
+      - ${OCS_CONFIG_DIR}:/config:ro
+    env_file:
+      - .env
+
+Your .env file should contain credentials for the InfluxDB v2 database::
+
+  INFLUXDB_V2_URL=http://influxdb:8086
+  INFLUXDB_V2_ORG=socs
```

Comment:
Let's make the example for 'org' consistently 'ocs'. 'socs' doesn't fit here. And users can set the org to something else if they'd like, but I imagine most won't care and will use whatever default we show.